dynamic_programming/number_coin_change/number_coin_change.go

```diff
@@ -0,0 +1,25 @@
+package main
+
```

Comment:
add a small description of the code at the top.

Reply:
duplicate of other comment. We can close it.